src/components/SignIn/SignIn.jsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import css from "./SignIn.module.css";
+import {customHistory} from "../../App";
+
+export class SignIn extends React.Component{
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            form: {
+                email: '',
+                password: ''
+            }
+        }
+    }
+
+    onChange = (event) => {
+      this.setState({
+          form: {
+              ...this.state.form,
+              [event.target.name]: event.target.value
+          }
+      })
+    };
+
+    onSubmit = (event) => {
+        if(event) {
+            event.preventDefault();
+        }
+
+        fetch('/auth/login', {
+            method: 'POST',
+            headers: {
+                'content-type': 'application/json'
+            },
+            body: JSON.stringify(this.state.form)
+        })
+            .then(response => response.json())
+            .then(response => {
+                if(!response.status){
```

Comment:
Here is already parsed json from response body. You can not have status here (except there is a field `status` in the json, but I'm not sure that's what you want). You can check status in the previous `.then`)

Reply:
fetch('/auth/register', {
          method: 'POST',
          headers: {
              'content-type': 'application/json'
          },
          body: JSON.stringify(this.state.form)
      })
          .then(response => {
              this.setState({
                  responseStatus: response.status
              });
              return response.json();
          })
          .then(response => {
              console.log(response);
              if(this.state.responseStatus === 200){
                  localStorage.setItem('TOKEN', response.accessToken);
                  this.props.onChangeFlag(true);
                  customHistory.push('/dashboard');
              } else {
                  console.log('ERROR: ', response);
              }
          })
          .catch(error => console.log('ERROR: ', error.message));
    };

Reply:
Just commit and push, PR will be updated

`if(this.state.responseStatus === 200)` - why do you take status from... `this`... why not from `response`?